### DISCUSSION TITLE
Capture sample only on trigger

### DISCUSSION THREAD

**Discussion by Ashram56** [gusmanb/logicanalyzer#58] [Ideas]
Hi there !

I've been using your logic analyzer SW, and one issue I encountered is that the protocol I'm trying to analyze has very long sequence where... nothing happens. Combined with the sample limit, I actually do not have a lot of meaningful data.

It would be really super useful if instead of starting capture at a given trigger and completely fill the buffer, you would basically trigger one sample acquisition for each trigger (ie the strobe trigger).

Is this already possible in the current implementation ?

Cheers

**Comment by gusmanb** [comment]
Hi!

Right now no, there is no such functionality, but that's something that when I get time I have planned to create, a "burst mode" where you specify how many samples per trigger you capture and how many burst you want (without exceeding the total buffer size).

For now I would say that the best you could do is to use a complex trigger that fires the capture when a concrete byte is received, so you will see the packet you're interested in.

Cheers.
